code/go/0chain.net/smartcontract/storagesc/validator.go

```diff
@@ -27,15 +27,15 @@ func getValidatorsList(balances c_state.StateContextI) (partitions.RandPartition
 	return all, nil
 }
 
-func (sc *StorageSmartContract) addValidator(t *transaction.Transaction, input []byte, balances c_state.StateContextI) (string, error) {
+func (ssc *StorageSmartContract) addValidator(t *transaction.Transaction, input []byte, balances c_state.StateContextI) (string, error) {
```

Comment:
Please avoid unrelated refactoring. Not only can it lead to hard to find errors and obscure the real changes being made, but it can also make merging the PR a nightmare.

I cannot see any real changes in validator.go, if so can you revert these changes.